### PR TITLE
Mark package org.gradle.composite.internal with NonNullApi

### PR DESCRIPTION
**Pull request by wolfs**
Split out of #25140.

### CODE REVIEW

File: subprojects/composite-builds/src/main/java/org/gradle/composite/internal/package-info.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2020 the original author or authors.
```
Comment (alllex) [inline subprojects/composite-builds/src/main/java/org/gradle/composite/internal/package-info.java:2]: 💄 should perhaps be 2023

**Review by alllex — APPROVED** [review]
Except for the tiny copyright suggestion, looks good!

File: subprojects/composite-builds/src/main/java/org/gradle/composite/internal/package-info.java (hunk 1 above)
Comment (wolfs) [reply subprojects/composite-builds/src/main/java/org/gradle/composite/internal/package-info.java]: Good catch, fixed.

**Comment by wolfs** [comment]
@bot-gradle test and merge

**Comment by wolfs** [comment]
@bot-gradle test and merge

**Pull request merged**